src/gui/element.rs

```diff
@@ -91,6 +92,48 @@ pub fn settings_container<'a>(
     // Data row for theme picker list.
     let theme_data_row = Row::new().push(left_spacer).push(theme_pick_list);
 
+    // Scale buttons for application scale factoring.
+    let (scale_title_row, scale_buttons_row) = {
+        let scale_title = Text::new("Scale").size(DEFAULT_FONT_SIZE);
```

Comment:
Should we write `UI Scale` here instead? Just to make it explicit what you are scaling.